src/components/input/input.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Create Elastic TextArea
+ *
+ * Adds an event listener to a `textarea` elment that responds to input events
+ * by either increasing or decreasing the `rows` attribute based on whether the
+ * `textarea` is scrolling or not. Returns an object containing a `destroy()`
+ * method to remove the event listener.
+ *
+ * 1. Sanity check to prevent an infinite loop if the textarea has a fixed height
+ *    @see https://github.com/cloudfour/cloudfour.com-patterns/pull/716#discussion_r428349375
```

Comment:
Oh, have we introduced a new pattern for documenting our JS? I would normally expect this comment to be inline as opposed to being included up here. 🤔 

It seems this pattern follows our CSS guidelines for adding comments. Sorry if I missed that we were changing our JS documentation style to follow the CSS documentation style. 😬 

FWIW, I have a preference lean toward having these types of comments inline within the body so I don't have to keep scrolling up/down to read and understand what this comment is referring to. 🙂

Reply:
That's fine — like you said, I'm used to this pattern from CSS-land when a comment is too long to go inline. I'll move it.